lib/streamlit/web/server/app_static_file_handler.py

```diff
@@ -64,7 +66,7 @@ def validate_absolute_path(self, root: str, absolute_path: str) -> str | None:
                 reason="File is too large",
```

Comment:
@kajarenc Looking at the [Tornado StaticFileHandler code](https://www.tornadoweb.org/en/stable/_modules/tornado/web.html#StaticFileHandler.validate_absolute_path), it seems like we do not need to check outside the static directory. I made sure we validate the realpath rather than the abs path. Let me know if you agree.